README.md

```diff
@@ -1,10 +1,10 @@
-# dmarc-import-terraform :cloud: :zap: #
+# dmarc-import-terraform #
 
-[![Build Status](https://travis-ci.com/cisagov/dmarc-import-terraform.svg?branch=develop)](https://travis-ci.com/cisagov/dmarc-import-terraform)
+[![GitHub Build Status](https://github.com/cisagov/dmarc-import-tf-module/workflows/build/badge.svg)](https://github.com/cisagov/dmarc-import-tf-module/actions)
 
-`dmarc-import-terraform` contains the Terraform configuration files
-that build the AWS infrastrusture used for parsing DMARC aggregate
-reports.  This repository goes along with
+`dmarc-import-terraform` contains the Terraform configuration files to
+build the AWS infrastrusture used for parsing DMARC aggregate reports.
```

Comment:
It's not your change, but `infrastructure` is misspelled here.

Reply:
See commit baf88ac.